src/utils/useTxMulti/useTxMulti.spec.ts

```diff
@@ -1,14 +1,14 @@
 import { renderHook, act } from '@testing-library/react-hooks';
 import isEmpty from 'ramda/src/isEmpty';
 import { fAccount, fNetwork } from '@fixtures';
+import { ITxObject, ITxToAddress, ITxValue, ITxData } from '@types';
 
 import { useTxMulti } from './useTxMulti';
-import { ITxObject } from '@types';
 
 const createTxRaw = (idx: number): Partial<ITxObject> => ({
-  to: 'address' + idx,
-  value: 'any',
-  data: 'empty'
+  to: ('address' + idx) as ITxToAddress,
+  value: 'any' as ITxValue,
+  data: 'empty' as ITxData
```

Comment:
How are these values valid?

Reply:
:shrug: i didn't write useTxMulti. I think its gotta do with useTxMulti not actually having any tx creation / validation logic in it. That is done outside of useTxMulti i think.